pkg/formats/influx/influx.go

```diff
@@ -579,8 +579,10 @@ func (f *InfluxFormat) fromSnmpInterfaceMetric(in *kt.JCHF) []InfluxData {
 			if speed, ok := ii["Speed"]; ok {
 				if ispeed, ok := speed.(int32); ok {
 					uptimeSpeed := in.CustomBigInt["Uptime"] * (int64(ispeed) * 10000) // Convert into bits here, from megabits. Also divide by 100 to convert uptime into seconds, from centi-seconds.
+					uptime_time := in.CustomBigInt["Uptime"]
 					if uptimeSpeed > 0 {
 						attrNew := util.CopyAttrForSnmp(attr, "IfInUtilization", kt.MetricInfo{Oid: "computed", Mib: "computed", Profile: profileName, Table: "if"}, f.lastMetadata[in.DeviceName], false)
+						attrNew_bitrate := util.CopyAttrForSnmp(attr, "IfInBitRate", kt.MetricInfo{Oid: "computed", Mib: "computed", Profile: profileName, Table: "if"}, f.lastMetadata[in.DeviceName], false)
```

Comment:
My only comment is to just use attrNew here for bitrate. No need to get a seperate map because whatever fields are valid for utilization will also be valid for bitrate (for a given interface).